chromadb/utils/embedding_functions.py

```diff
@@ -665,6 +666,9 @@ def __init__(
         project_id: str = "cloud-large-language-models",
```

Comment:
Is project_id specific to the user deployment or can this `cloud-large-language-models` be used as default? Reading the docs makes me think this cannot be defaulted and must always be supplied. If so let's add a check for this and remove the default value.

Reply:
Yes, project_id is a parameter that have to be supplied. It is used along with corresponding project name in google service account. So it should be removed from default value.